azure/scope/cluster.go

```diff
@@ -775,6 +796,9 @@ func (s *ClusterScope) FailureDomains() []string {
 		fds[i] = id
 		i++
 	}
+
+	sort.Strings(fds)
```

Comment:
Looks like we have UT coverage for this new enforcement of a sorted string response, thanks!